src/Tests/ObjectLifetimeTests/ObjectLifetimeTests.Wux/ObjectLifetimeTests.Wux.csproj

```diff
@@ -0,0 +1,224 @@
+    </Compile>
+    <Compile Include="..\Tests\RealMainPage.xaml.cs">
+      <Link>Tests\RealMainPage.xaml.cs</Link>
+      <DependentUpon>..\Tests\RealMainPage.xaml</DependentUpon>
+    </Compile>
+    <Compile Include="App.xaml.cs">
+      <DependentUpon>App.xaml</DependentUpon>
+    </Compile>
+    <Compile Include="..\Tests\ClrObjects.cs">
+      <Link>Tests\ClrObjects.cs</Link>
+    </Compile>
+    <Compile Include="..\Tests\CollectionExtensions.cs">
+      <Link>Tests\CollectionExtensions.cs</Link>
+    </Compile>
+    <Compile Include="..\Tests\CustomClrObject.cs">
+      <Link>Tests\CustomClrObject.cs</Link>
+    </Compile>
+    <Compile Include="..\Tests\CustomControl.cs">
+      <Link>Tests\CustomControl.cs</Link>
+    </Compile>
+    <Compile Include="..\Tests\CustomDependencyObject.cs">
+      <Link>Tests\CustomDependencyObject.cs</Link>
+    </Compile>
+    <Compile Include="..\Tests\CustomGroupedItemPages.cs">
+      <Link>Tests\CustomGroupedItemPages.cs</Link>
+    </Compile>
+    <Compile Include="..\Tests\CycleTestCanvas.cs">
+      <Link>Tests\CycleTestCanvas.cs</Link>
+    </Compile>
+    <Compile Include="MainPage.xaml.cs">
+      <DependentUpon>MainPage.xaml</DependentUpon>
+    </Compile>
+    <Compile Include="..\Tests\ObjectLifetimeTests.cs">
+      <Link>Tests\ObjectLifetimeTests.cs</Link>
+    </Compile>
+    <Compile Include="Properties\AssemblyInfo.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <AppxManifest Include="Package.appxmanifest">
+      <SubType>Designer</SubType>
+    </AppxManifest>
+  </ItemGroup>
+  <ItemGroup>
+    <Content Include="Properties\Default.rd.xml" />
+    <Content Include="Assets\LockScreenLogo.scale-200.png" />
+    <Content Include="Assets\SplashScreen.scale-200.png" />
+    <Content Include="Assets\Square150x150Logo.scale-200.png" />
+    <Content Include="Assets\Square44x44Logo.scale-200.png" />
+    <Content Include="Assets\Square44x44Logo.targetsize-24_altform-unplated.png" />
+    <Content Include="Assets\StoreLogo.png" />
+    <Content Include="Assets\Wide310x150Logo.scale-200.png" />
+  </ItemGroup>
+  <ItemGroup>
+    <ApplicationDefinition Include="App.xaml">
+      <Generator>MSBuild:Compile</Generator>
+      <SubType>Designer</SubType>
+    </ApplicationDefinition>
+    <Page Include="..\Tests\MyControl.xaml">
+      <Link>Tests\MyControl.xaml</Link>
+      <Generator>MSBuild:Compile</Generator>
+      <SubType>Designer</SubType>
+    </Page>
+    <Page Include="..\Tests\ObjectLifetimePage.xaml">
+      <Link>Tests\ObjectLifetimePage.xaml</Link>
+      <Generator>MSBuild:Compile</Generator>
+      <SubType>Designer</SubType>
+    </Page>
+    <Page Include="..\Tests\RealMainPage.xaml">
+      <Link>Tests\RealMainPage.xaml</Link>
+      <Generator>MSBuild:Compile</Generator>
+      <SubType>Designer</SubType>
+    </Page>
+    <Page Include="MainPage.xaml">
+      <Generator>MSBuild:Compile</Generator>
+      <SubType>Designer</SubType>
+    </Page>
+  </ItemGroup>
+  <ItemGroup>
```

Comment:
Do adding WUX lifetime tests make sense here?

Reply:
we should open a separate issue for additional tests, as there are dependencies on this PR

Reply:
This comment wasn't regarding adding additional tests, but rather this file is for tests that are testing WUX, not MUX.